js/pages/talent.js

```diff
@@ -52,9 +52,7 @@ function submitNewTalent() {
   const submitData = {
     name: DOMElems.name.value,
     budget: DOMElems.budget.value,
-    talentTeam: {
```

Comment:
Also here, merge conflict seems incorrectly resolved, if I remember correctly @cornelisdemooij  changed it to this yesterday.